docs/howto/manage-web-app-charms/use-web-app-charm.rst

```diff
@@ -347,3 +347,52 @@ The Pebble logs are available via Grafana or Loki and can be viewed in
 the **WebApp Operator** dashboard for Flask and Django.
 For other frameworks, you may access the logs by picking ``loki`` in the
 ``http://<IP_ADDRESS>/<JUJU_MODEL_NAME>-grafana/explore`` page.
+
+
+Configure TLS for a web app charm
+---------------------------------
+
+You can configure TLS if your ingress provider charm supports integration with
+a certificate provider charm. To test locally, you can use the
```

Comment:
> To test...

By saying "test" here in the first paragraph, we're making what looks like another task statement that contradicts the heading. Is the user going to _configure_ TLS, or _test_ with a charm locally?

Instead, I think we ought to say that the best way to test the TLS connection is with that charm.